publication/test/support/fixtures/seed_project/project_seed.ex

```diff
@@ -98,7 +99,7 @@ defmodule FieldPublication.Test.ProjectSeed do
 
     {:ok, %FieldPublication.DatabaseSchema.Publication{} = publication} =
       Publications.put(publication, %{
-        "publication_date" => Date.utc_today(),
+        "publication_date" => Date.from_iso8601!("2024-06-05"),
```

Comment:
Is this still necessary? 

Actually I am not sure why the old line was here in the first place, the date should have been set in `Publications.create_from_replication_input/1`. With your change, this should be set using the default value instead.

Reply:
Well, I tried and it works just fine without this line. I removed it and will merge that.

Reply:
Was necessary after all, commit #[97d8520](https://github.com/dainst/idai-field/pull/256/commits/97d8520034a0c68a1682265838da344bc2f53a77) is reverted.